runelite-client/src/main/java/net/runelite/client/plugins/driftnet/DriftNetConfig.java

```diff
@@ -76,12 +76,12 @@ default boolean highlightUntaggedFish()
 	)
 	@Range(
 		min = 1,
-		max = 60
+		max = 100
 	)
 	@Units(Units.TICKS)
 	default int timeoutDelay()
 	{
-		return 10;
+		return 100;
```

Comment:
60 seconds is a very long time, i might have undershot the correct time with 6 seconds but what's the motivation for a tenfold increase?